.github/workflows/release.yml

```diff
@@ -28,12 +34,13 @@ jobs:
 
       - name: Build, test and push
         run: |
-          echo ${{ secrets.DOCKERHUB_PASSWORD }} | docker login -u ${{ secrets.DOCKERHUB_USER }} --password-stdin
-          docker build -t ppiper/node-browsers:${{ env.PIPER_version }} .
-          docker build -t ppiper/node-browsers:${{ env.PIPER_version }}-node12 node12
-          docker push ppiper/node-browsers:${{ env.PIPER_version }}
-          docker push ppiper/node-browsers:${{ env.PIPER_version }}-node12
-
+          echo ${{ secrets.DH_PW }} | docker login -u ${{ secrets.DH_USER }} --password-stdin
+          docker build --build-arg=BASE_IMAGE=node:${{ matrix.base-image }} --tag ppiper/node-browsers:${{ env.PIPER_version }}-${{ matrix.base-image }} .
+          docker push ppiper/node-browsers:${{ env.PIPER_version }}-${{ matrix.base-image }}
+          if [ "${{ matrix.base-image }}" == 10-buster ]; then
```

Comment:
I think when you move this to a dedicated stage we can use step condition to just run on node:10. That may be a bit cleaner.